Tests/Marketplace/upload_packs.py

```diff
@@ -1053,7 +1028,7 @@ def main():
     extract_destination_path = option.extract_path
     storage_bucket_name = option.bucket_name
     service_account = option.service_account
-    target_packs = option.pack_names if option.pack_names else ""
+    packs_to_upload = option.pack_names or ""
```

Comment:
```suggestion
    modified_packs_to_upload = option.pack_names or ""
```
helps understanding why we upload them (please change in pack_names_to_upload too)